ipmininet/install/build_openr-rc-20190419-11514.sh

```diff
@@ -0,0 +1,301 @@
+#!/bin/bash
```

Comment:
Did you write this install script yourself, or was it generated by some of the OpenR build-tools?

How "portable" across OpenR release should this be?

Reply:
I generated the basic script from https://github.com/facebook/openr/tree/master/build#build-on-debian-host-without-docker.
The 'vanilla' script can be found in 622f6109bda4b91fd76adb577c175fef181182d6.
The following manual adjustments to the build script mainly pin dependencies for the release.

I would like to make it more portable. However, it seems that there is some re-work going on for the build chain and the documented build process is very brittle at the moment. I'm keeping an eye on the development and I will adopt the 'official' build instructions as soon as they have been stabilized.